nextflow.config

```diff
@@ -9,12 +9,14 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
-    input                      = null
+    input_sample                = null
+    input_cycle                 = null
+    marker_sheet                = null
+    illumination                = null
 
     // Illumination options
-    illumination               = false
+    // illumination               = false
```

Comment:
```suggestion
```

Reply:
as it's already in your input options section

Reply:
Yes this can be cleaned up to have a single `= null` entry and no commented-out instance, but I would still like to keep the header comment as there will be more options in this section soon.